### DISCUSSION TITLE
Flame Head

### DISCUSSION THREAD

**Discussion by bigdaddy2142** [pierrelasse/YimStuff#15] [Ideas]
You have the best Yim addon menu. Could you possibly make an option to have constant fire coming from the head similar to ghost rider mod? 
![aae6fe-GHOSTRIDER2](https://github.com/pierrelasse/YimStuff/assets/153862933/09659ec8-ae16-4ab4-9982-e70e62881fb3)

**Comment by pierrelasse** [comment]
Thanks.

Could you maybe provide me a script where this was done?
Or the ptfx assets things.
